server/sonar-server/src/main/java/org/sonar/server/es/IndexingResult.java

```diff
@@ -60,13 +60,8 @@ public long getSuccess() {
     return successes;
   }
 
-  /**
-   * Get the failure ratio,
-   * if the total is 0, we always return 1 in order to break loop
-   * @see {@link RecoveryIndexer#recover()}
-   */
-  public double getFailureRatio() {
-    return total.get() == 0 ? 1 : ((1.0d * getFailures()) / total.get());
+  public double getSuccessRatio() {
+    return total.get() == 0 ? 1.0 : (1.0 * successes) / total.get();
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Add parentheses to make the operator precedence explicit. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3AS864)